admin/class-boldgrid-backup-admin-dashboard.php

```diff
@@ -79,6 +79,43 @@ public function get_cards() {
 	 * @since 1.11.0
 	 */
 	public function page() {
+		add_thickbox();
```

Comment:
Not sure exactly where would be a good place for that since the page() callbacks for the various pages are not all located in the same place ( some are in seperate files, some are in core, etc. ) . It may work if we hook the add_thickbox() to the load-(page) action for each page in a loop in boldgrid-backup.php

Reply:
I added a function in the admin_core, called add_thickbox, and then made the call to it using admin_enqueue_scripts hook in the add hooks section of boldgrid_backup.php . It causes the add_thickbox() function to be run on any admin screen that has the "boldgrid-backup" string in it.